interp/notation.ml

```diff
@@ -2103,18 +2113,30 @@ let split_notation_string str =
     if i < String.length str then
       if str.[i] == ' ' then
         push_token beg i (loop_on_whitespace (i+1) (i+1))
+      else if beg = i && str.[i] = '"' then
+        loop_on_string i (i+1)
       else
         loop beg (i+1)
     else
       push_token beg i []
   and loop_on_whitespace beg i =
     if i < String.length str then
       if str.[i] != ' ' then
-        push_whitespace beg i (loop i (i+1))
+        push_whitespace beg i (loop i i)
```

Comment:
What is the significance of this change?

Reply:
Because the non-space character at position i can now be a `"`, so we need not to skip it so that the test on `beg = i && str.[i] = '"' ` applies to it.